ldm/invoke/CLI.py

```diff
@@ -487,6 +492,19 @@ def do_command(command:str, gen, opt:Args, completer) -> tuple:
         command = '-h'
     return command, operation
 
+def concepts_to_triggers(gen, prompt:str)->str:
+    concepts = re.findall('<([^>]+)>',prompt)
```

Comment:
this regex causes a huge security hole

`invoke> "a <my arbitrary set of words here>" -s 2 -t`

causes an attempt to build a URL with path `/sd-concepts-library/big thing with a big gap/resolve/main/README.md`

the regex should be far more restrictive (it should probably only match `A-Za-Z0-0_\-`, explicitly typed out like that)

also it should be a constant to avoid copy/pasting.

Reply:
There are actually two very similar-looking but different regexps here, but point taken. I've restricted the inner pattern to [\w-] (which should be equivalent to the pattern you recommended), and store the precompiled patterns into attributes of the object at init time.